corehq/util/view_utils.py

```diff
@@ -69,7 +70,7 @@ def inner(request, *args, **kwargs):
         except BadRequest as e:
             return _get_json_exception_response(400, request, e)
         except Exception as e:
-            notify_exception(request, 'JSON exception response: {}'.format(e))
+            notify_exception(request, 'JSON exception response: {}'.format(smart_bytes(e)))
```

Comment:
Agreed. I feel this is a pragmatic solution given our situation. (Is that a Pink Floyd reference?! "Hey you, standing in the ... aisles"? ... Don't help them to bury the light, @millerdev. Don't give in without a fight! :laughing:)